### DISCUSSION TITLE
Login - Page bloquée

### DISCUSSION THREAD

**Discussion by JarodStreck** [web-classroom/web-classroom.github.io#19] [Q&A : Pratique / Labos] [answered]
![image](https://github.com/web-classroom/web-classroom.github.io/assets/38767523/88754726-965f-4b2a-aa3e-d52d12aeeb60)

Lorsqu'on essaie de se connecter avec un faux mot de passe et qu'on est connecté au réseau de l'école, on se fait bloquer par leur firwall.

**Comment by OlivierLmr** [comment, marked as answer]
Merci de m'en avoir informé !

Je vais prendre contact avec le service informatique à ce sujet. En attendant, vous pouvez skip ce flag, et dans le pire des cas je lancerai une autre VM au moment du labo de la semaine prochaine.
